pkg/restapi/rp/operation/operations_test.go

```diff
@@ -141,6 +141,92 @@ func TestVerifyVC(t *testing.T) {
 	})
 }
 
+func TestVerifyVP(t *testing.T) {
+	t.Run("test error from parse form", func(t *testing.T) {
+		svc := New(&Config{})
+		svc.client = &mockHTTPClient{postErr: fmt.Errorf("post error")}
+
+		rr := httptest.NewRecorder()
+		svc.verifyVP(rr, &http.Request{Method: http.MethodPost})
+		require.Equal(t, http.StatusInternalServerError, rr.Code)
+		require.Contains(t, rr.Body.String(), "failed to parse form")
```

Comment:
Are these two tests the same?

Reply:
No actually both are different, when is failing the test due to missing form body and other one have a invalid form body. I have added the exact error its throwing to be more specific and avoid any confusion.